charts/keybase-alertmanager/Chart.yaml

```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for Kubernetes
 name: keybase-alertmanager
-version: 0.1.0
+version: 1.1.0
```

Comment:
that's not how semver works

Reply:
true that, but we it will be the second of our releases. the first one was done when creating the helm repository on github pages. though, we can remove the first release (1.0.0) and label all the changes we've done thee as the version 1.0.0